docs/index.mdx

```diff
@@ -3,59 +3,55 @@ title: Introduction
 description: Welcome to the Next.js Documentation.
 related:
   title: Next Steps
-  description: Get started with Next.js by following the installation guide.
+  description: Create your first application and learn the core Next.js features.
   links:
-    - app/getting-started/installation
+    - app/getting-started
 ---
 
 Welcome to the Next.js documentation!
 
 ## What is Next.js?
 
-Next.js is a React framework for building full-stack web applications. You use React Components to build user interfaces, and Next.js for additional features and optimizations.
+Next.js is a React framework for building full-stack web applications. You use React Components to build user interfaces, and Next.js for additional features and optimizations like routing, server rendering, and more.
```

Comment:
```suggestion
Next.js is a React framework for building full-stack web applications. You use React Components to build user interfaces, and Next.js for additional features and optimizations.
```

For optimizations I was thinking more like images, fonts, scripts, etc – but my main there here is not wanting to drop too many terms, too early.